frontend/pages/campaign/add.vue

```diff
@@ -1,22 +1,335 @@
+              Continue
+            </b-button>
+          </div>
+        </div>
+        <hr>
+        <div>
+          <h4 class="title is-4">
+            Import Existing Campaign
+          </h4>
+          <h4 class="subtitle is-6">
+            If you have an existing campaign contract, you may import it directly without keying in the details above.
+          </h4>
+          <b-field label="External Campaign Address">
+            <b-input disabled />
+          </b-field>
+          <b-button type="is-success" disabled>
+            Continue
+          </b-button>
+        </div>
+      </b-step-item>
+      <b-step-item step="1" label="Basic Info" :clickable="false">
+        <div>
+          <h1 class="title has-text-centered">
+            Basic Information
+          </h1>
+          <h1 class="subtitle is-6 has-text-centered">
+            Let everyone know what you are trying to raise funds for.
+          </h1>
+        </div>
+        <div class="block my-5">
+          <h6 class="title is-5">
+            Required Details
+          </h6>
+          <b-field label="Campaign Name" horizontal>
+            <b-input v-model="campaignName" placeholder="Fund research on reducing micoplastics in our oceans!" />
+          </b-field>
+          <b-field label="Beneficiary Address" horizontal>
+            <b-input disabled :value="beneficiaryAddress" />
+          </b-field>
+          <b-field label="Beneficiary URL" horizontal>
+            <b-input v-model="beneficiaryURL" type="url" placeholder="https://example.com" />
+          </b-field>
+          <b-field label="Description" horizontal>
+            <b-input v-model="campaignDescription" type="textarea" />
+          </b-field>
+          <hr>
+          <h6 class="title is-5">
+            Optional Details
+          </h6>
+          <b-field label="Target Amount" horizontal grouped>
+            <b-input v-model="targetAmount" type="number" step="any" min="0" />
+            <p class="control">
+              <span class="button is-static">ETH</span>
+            </p>
+          </b-field>
+          <b-field label="End Date" horizontal>
+            <b-datepicker
+              v-model="endDate"
+              placeholder="Click to select..."
+              :min-date="new Date()"
+            >
+              <b-button
+                label="Clear"
+                type="is-danger"
+                icon-left="close"
+                outlined
+                @click="endDate = null"
+              />
+            </b-datepicker>
+          </b-field>
+        </div>
+        <b-button @click="moveBack">
+          Back
+        </b-button>
+        <b-button type="is-success" :disabled="!isValidated" @click="moveToReview">
+          Next
+        </b-button>
+      </b-step-item>
+
+      <b-step-item step="2" label="Review & Deploy" :clickable="false">
+        <div>
+          <h1 class="title has-text-centered">
+            Review & Deploy
```

Comment:
Might want to use html codes for special characters

Reply:
Changed!